src/Features/CSharp/Portable/InlineHints/CSharpInlineParameterNameHintsService.cs

```diff
@@ -73,17 +74,39 @@ public CSharpInlineParameterNameHintsService(IGlobalOptionService globalOptions)
             BaseArgumentListSyntax argumentList,
             CancellationToken cancellationToken)
         {
+            var parametersOfNamedArguments = GetParametersOfNamedArguments(semanticModel, argumentList, cancellationToken);
             foreach (var argument in argumentList.Arguments)
             {
                 if (argument.NameColon != null)
                     continue;
 
                 var parameter = argument.DetermineParameter(semanticModel, cancellationToken: cancellationToken);
+
+                if (parametersOfNamedArguments.Contains(parameter))
```

Comment:
Aren't we already skipping here because of line 80? 

```csharp
if (argument.NameColon != null)
    continue;
```

Reply:
This is for this case:
![image](https://user-images.githubusercontent.com/40616383/158450632-b16484b6-abe0-4292-97d1-e5e9845fd12b.png)

In which DetermineParameter maps the argument "true" to the parameter "d". So GetParametersOfNamedArguments goes through and gets the parameters in those mappings and makes sure to skip them. The argument.NameColon is necessary for the named argument "d:" since we still want to skip it.

Reply:
ahhhh.... I get it now. Thanks for the explanation